experimental/sql/test/frontend/group_by.ibis

```diff
@@ -0,0 +1,37 @@
+# RUN: rel_opt.py -f ibis %s | filecheck %s
+
+table = ibis.table([("a", "string"), ("b", "int64"), ("c", "int64")], 't')
+table.group_by([table.a, table.b]).aggregate(table.c.sum().name('c_sum'))
```

Comment:
Does the "empty" aggregation (i.e., group-by without aggregation) work? Can you add a test case for it?

Reply:
I have removed the GroupedTable handling from the prototype since TPC-H queries do not need this (AFAIK). Hence, this will not work anymore.

I can add it again, but I feel that we will throw away (most of) this prototype after the thesis submission and go with the optimized design instead.

Reply:
And even if we want this behavior, we could still express it with the current version, by using `.aggregate()`, right?

Reply:
Yes, true. We can produce an empty aggregation. We will fail to compile some ibis constructs that are valid for other backends, though, no? But I guess that's OK; just document it.